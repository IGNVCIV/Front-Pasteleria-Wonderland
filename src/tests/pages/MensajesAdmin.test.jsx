import '@testing-library/jest-dom';
import { render, screen } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import MensajesSection from "../../components/MensajesAdmin";

describe("Componente MensajesAdmin", () => {
  it("se renderiza correctamente con el título y mensaje vacío", () => {
    render(<MensajesSection />);

    expect(screen.getByText("Bandeja de Mensajes")).toBeInTheDocument();

 
    const mensaje = screen.getByText("No hay mensajes nuevos");
    expect(mensaje).toBeInTheDocument();
    expect(mensaje).toHaveClass("mensaje-vacio");
  });

  it("coincide con el snapshot actual", () => {
    const { container } = render(<MensajesSection />);
    expect(container).toMatchSnapshot();
  });
});


