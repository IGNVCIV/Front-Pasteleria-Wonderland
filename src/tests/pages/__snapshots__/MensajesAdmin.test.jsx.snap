// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Componente MensajesAdmin > coincide con el snapshot actual 1`] = `
<div>
  <section
    class="mensajes-section"
  >
    <h2>
      Bandeja de Mensajes
    </h2>
    <p
      class="mensaje-vacio"
    >
      No hay mensajes nuevos
    </p>
  </section>
</div>
`;
